sf-r53-update

```diff
@@ -93,6 +93,13 @@ unless ENV.key?('AWS_REGION')
 
 end
 
+unless options[:noop]
```

Comment:
Would it be desirable to also have a parameter to skip the delay? If you're only updating a single object than the sleep would be unnecessary?

Reply:
I'll add that

Reply:
Done. The delay's not part of the configuration.